src/vs/editor/contrib/codeAction/browser/codeActionModel.ts

```diff
@@ -234,10 +234,15 @@ export class CodeActionModel extends Disposable {
 
 						// Search for quickfixes in the curret code action set.
 						const foundQuickfix = codeActionSet.validActions?.some(action => action.action.kind ? CodeActionKind.QuickFix.contains(new CodeActionKind(action.action.kind)) : false);
-
-						if (!foundQuickfix) {
-							const allMarkers = this._markerService.read({ resource: model.uri });
-
+						const allMarkers = this._markerService.read({ resource: model.uri });
+						if (foundQuickfix) {
+							for (const action of codeActionSet.validActions) {
+								if (action.action.command?.arguments?.some(arg => typeof arg === 'string' && arg.includes('_typescript.applyFixAllCodeAction'))) {
```

Comment:
Is `_typescript.applyFixAllCodeAction` available as a const somewhere?

Reply:
it's not atm - it's used here tho https://github.com/microsoft/vscode/blob/3822928d403f8fc095f04ec7028a09f1d37db8e9/extensions/typescript-language-features/src/languageFeatures/quickFix.ts#L66

since we use it twice, alternatively could just make it a const above?